llvm/test/CodeGen/NVPTX/nvptx-debug-symbol-name.ll

```diff
@@ -0,0 +1,33 @@
+; RUN: llc -mtriple=nvptx64-nvidia-cuda -mcpu=sm_86 < %s | FileCheck %s
```

Comment:
```suggestion
; RUN: llc -mtriple=nvptx64-nvidia-cuda -mcpu=sm_86 < %s | FileCheck %s
; RUN: %if ptxas %{ llc < %s -march=nvptx64 -mcpu=sm_68 -verify-machineinstrs | %ptxas-verify %}
```

Reply:
This should work, if the user has `ptxas` we want to make sure the emitted debug symbols are still valid.